---
int16 pos_x
int16 pos_y
